product/models.py

```diff
@@ -55,17 +79,13 @@ class ThemeProduct(models.Model):
     product = models.ForeignKey(Product, on_delete = models.SET_NULL, null=True)
 
     class Meta:
-        db_table = 'themeproducts'
+        db_table = 'theme_products'
 
 class Review(models.Model):
-    product    = models.ForeignKey('Product', on_delete = models.SET_NULL, null=True)
-    user       = models.ForeignKey('User', on_delete = models.SET_NULL, null=True)
+    product    = models.ForeignKey(Product, on_delete = models.SET_NULL, null=True)
+    user       = models.ForeignKey(User, on_delete = models.SET_NULL, null=True)
     order      = models.ForeignKey('Order', on_delete = models.SET_NULL, null=True)
     rate       = models.IntegerField(null=True)
     content    = models.TextField(null=True)
-    image      = models.URLField(null=True)
+    image_url  = models.URLField(max_length = 2000, null=True)
     created_at = models.DateTimeField(auto_now = True)
```

Comment:
다른 모델과 같이 테이블명 추가해주세요
```suggestion
    class Meta:	
        db_table = 'reviews'
```

Reply:
수정했습니다 :)